net-im/psimedia/psimedia-9999.ebuild

```diff
@@ -29,13 +29,15 @@ RDEPEND="${DEPEND}
 	media-plugins/gst-plugins-jpeg:1.0
 	media-plugins/gst-plugins-opus:1.0
 	media-plugins/gst-plugins-v4l2:1.0
-	~net-im/psi-${PV}[extras?]
+	psi? ( ~net-im/psi-${PV}[extras?] )
 "
+# and optional media-plugins/gst-plugins-webrtcdsp:1.0 for echo cancellation
 
 src_configure() {
 	local mycmakeargs=(
-		-DUSE_PSI=$(usex extras 0 1)
-		-DBUILD_DEMO=0
+		-DUSE_PSI=$(usex extras)
+		-DBUILD_DEMO=$(usex demo)
+		-DBUILD_PSIPLUGIN=$(usex psi)
```

Comment:
Looks to me like `PSI` and `PSIPLUGIN` should be enabled with a single USE flag. So maybe ditch `psi` and keep using `extras`? Or the other way around.

I'd even enable `demo` when using `extras`.

Reply:
Not quite. Those are different things
extras - build either for Psi or Psi+ (for net-im/psi it's controlled by `extras` flag)
psi - where at all to build the Psi/Psi+ plugin (there is also generic/psi-independent plugin).

Maybe it's somewhat overcomplicated. Maybe it's easier to drop Psi+ brand together with extras flag as well as drop `psi` flag here and always built this Psi plugin while no other users of psimedia exist.

Reply:
Yeah that sounds simpler.

Reply:
any update here?